v2/python/sample.py

```diff
@@ -39,7 +39,7 @@
 TOKEN_SECRET = None
 
 
-def request(host, path, url_params=None):
+def request(host, path, url_params={}):
```

Comment:
This looks like a common Python syntax pitfall, mutable types in function argument defaults:
* http://effbot.org/zone/default-values.htm

The "None up top, empty list/dict/whatever inside" is the best practice for this situation, leaving it as-is.